assets/js/modules/adsense/datastore/clients.js

```diff
@@ -155,6 +157,40 @@ const baseSelectors = {
 
 		return clients[ accountID ];
 	},
+
+	/**
+	 * Gets the AdSense For Content (AFC) client for the given AdSense account.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state     Data store's state.
+	 * @param {string} accountID The AdSense Account ID to fetch AFC client for.
+	 * @return {(Object|undefined)} An AdSense client; `undefined` if not loaded.
+	 */
+	getAFCClient: createRegistrySelector(
+		( select ) => ( state, accountID ) => {
+			if ( undefined === accountID ) {
+				return undefined;
+			}
+
+			const clients = select( MODULES_ADSENSE ).getClients( accountID );
+
+			if ( clients === undefined ) {
+				return undefined;
+			}
+
+			const afcClients = clients.filter( ( client ) => {
+				return 'AFC' === client.productCode;
+			} );
+
+			if ( ! afcClients.length ) {
+				return undefined;
```

Comment:
This should return `null` instead. At this point, nothing is loading, there is just no AFC client.

Reply:
👍